packages/ods-react/src/components/pagination/src/components/PaginationButtonWithTooltip/PaginationButtonWithTooltip.tsx

```diff
@@ -0,0 +1,17 @@
+import type { FC, ReactNode } from 'react';
+import { Tooltip, TooltipContent, TooltipTrigger } from '../../../../tooltip/src';
+
+const PaginationButtonWithTooltip: FC<{
```

Comment:
declare prop in a separate interface and export as type like others (even if it's not publicly exposed)